src/Workspaces/Remote/Core/Services/RemoteExperimentationService.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using System.Threading;
+using Microsoft.CodeAnalysis.Experiments;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.Remote.Services
+{
+    [ExportWorkspaceService(typeof(IExperimentationService), ServiceLayer.Host), Shared]
```

Comment:
more of an understanding question - both this one and the VisualStudioExperimentationService are exported at the host layer - could that cause problems?  I'm guessing no since the VS one would be in-proc only.

Reply:
There's two separate MEF compositions, so the one here doesn't get picked up in VS and vice versa. Correct @heejaechang?

Reply:
yep.